android_webview/test/shell/src/org/chromium/android_webview/shell/AwShellActivity.java

```diff
@@ -420,18 +420,18 @@ public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
         // If the device is not supported, show an alert and quit.
-        if (!SupportedDevices.isSupported(this)) 
-        {
-            createAlertDialog(AwShellActivity.this, "ERROR", "This device is not currently supported by WebARonARCore.", new DialogInterface.OnClickListener()
-                {
-                    @Override
-                    public void onClick(DialogInterface dialog, int which)
-                    {
-                        finish();
-                    }
-                }, 1, "Ok", null, null).show();
-            return;
-        }
+        // if (!SupportedDevices.isSupported(this)) 
+        // {
+        //     createAlertDialog(AwShellActivity.this, "ERROR", "This device is not currently supported by WebARonARCore.", new DialogInterface.OnClickListener()
```

Comment:
may I recommend that we still do a warning that your device *may* not be supported in this case, but still let them continue? Up to you - LGTM.

Reply:
Not sure about it if we are still going to allow the app to continue. In the list of DP2 supported devices there are more devices supported and those will still get the "annoying" warning dialog... not 100% sure but my take for now is that if we get rid of the check and allow all the apps to run on any device, so be it :(